### PR TITLE
[MM-37157, MM-37159] Add new branding graphics welcome to MM, tips & next steps and shared background

### PR DESCRIPTION
**Pull request by deanwhillier**
#### Summary
Updates the "download apps" graphic and background graphic (MM-37159), both with theming. Also updates the graphic for the "Welcome to Mattermost" screen that precedes tips and next steps (MM-37157). Background update is shared by both tickets

The background uses a `<symbol/>` of the pill shape and surrounding 'stipple' and is instanced for each pill visible in the background with appropriate transforms to rotate and position. The 'stipple' is made with an inlined JPG (base64) used as a mask on a `<rect />` to allow for theming while keeping the overall size a fraction of what it would be if the `stipple` was vector-based.

In talking with @matthewbirtch, it was decided that the background would be anchored to the top left and would not scale up past "100%". It does scale down to a min of "50%" however, otherwise the graphic gets lost on smaller screens ... can adjust as needed.

This PR also makes a small tweak to scrolling to remove the permanently visible scrollbar. There was a recent fix for [a bug](https://mattermost.atlassian.net/browse/MM-37065) where scrolling was not working on iOS. The fix uses `-webkit-overflow-scrolling: touch;` and `overflow-y: scroll;` to fix the scrolling issue. Seems that only the `-webkit-overflow-scrolling: touch;` is needed to fix the iOS bug and the other can remain as `overflow-y: auto;` so the scrollbar only shows when there is content to be scrolled.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37159
https://mattermost.atlassian.net/browse/MM-37157

### Related Tickets
https://mattermost.atlassian.net/browse/MM-37065

#### Screenshots
image

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by andrewbrown00 — CHANGES_REQUESTED** [review]
Thanks, @deanwhillier, small change:

- [ ] Remove the **Download the Desktop and Mobile Apps** section from the **Welcome to Mattermost** screen. 
   - It should only show on the **Tips & Next Steps** screen.
   image

**Comment by andrewbrown00** [comment]
@deanwhillier disregard the feedback, looks like there was an update to add the section https://github.com/mattermost/mattermost-webapp/pull/8523, it wasn't updated in the design file.

> Thanks, @deanwhillier, small change:
> 
> * [ ]  Remove the **Download the Desktop and Mobile Apps** section from the **Welcome to Mattermost** screen.
>   
>   * It should only show on the **Tips & Next Steps** screen.
>     image

**Review by andrewbrown00 — APPROVED** [review]
Thanks, @deanwhillier, changes look great 🙂 

* Verified backgrounds and illustrations updates align with new branding
* Verified the colors are correct when switching themes

Getting Started | Tips & Next Steps
---------- | ----------
image | image
image | image

File: components/next_steps_view/images/onboarding-bg-svg.tsx [hunk 1]
```diff
@@ -2,919 +2,50 @@
...
-                    ry='3.10334'
-                    transform='matrix(0.706583 -0.70763 0.706583 0.70763 124.735 204.233)'
-                    fill='var(--button-bg)'
-                />
-                <circle
-                    r='3.10335'
-                    transform='matrix(0.706583 -0.70763 0.706583 0.70763 146.662 226.193)'
-                    fill='var(--button-bg)'
-                />
+            <g style={{transformOrigin: 'top left', transform: 'translate(184px, -772px) rotate(45deg)'}}>
```
Comment (michelengelen) [inline components/next_steps_view/images/onboarding-bg-svg.tsx:43]: nice improvement! (to all of the groups)

**Review by michelengelen — APPROVED** [review]
really nice ... LGTM! :sparkles:

**Review by esethna — APPROVED** [review]

**Review by nevyangelova — APPROVED** [review]
LGTM

**Review by jgilliam17 — APPROVED** [review]
Thanks @deanwhillier 
Tested, looks good to merge.
- Verified new branding graphics and background for Welcome and Tips&Next Steps screens - looks good in both light and dark themes.

**Pull request merged**